docs/src/main/sphinx/connector/druid.rst

```diff
@@ -41,3 +41,19 @@ secured by basic authentication by updating the URL and adding credentials:
 
 Now you can access your Druid database in Trino with the ``druiddb`` catalog
 name from the properties file.
+
+.. _druid-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in the
+Druid broker. In addition to the :ref:`globally available
```

Comment:
AFAIK should be "Druid database". The broker is just a coordinator, doesn't actually have the data itself.